src/Graph5/Graph5.tsx

```diff
@@ -0,0 +1,46 @@
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import data from "./DataGraph5.json";
+
+function findHighestUV(data: { name: string; uv: number }[]): number {
+	const uvValues = data.map((item) => item.uv);
+	const highestUV = Math.max(...uvValues);
+	return highestUV;
+}
+
+const highestUVValue = findHighestUV(data);
+
+export function Graph5() {
+	return (
+		<div>
+			<h1 className="text-2xl  font-bold">{highestUVValue}</h1>
+			<span className="bg-teal-400 h-2 w-2 rounded-full inline-block"></span>
```

Comment:
Zrob se to jako div, nie bedziesz musial kombinowac wtedy z inline-block, semantycznie tez bedzie lepiej, span to bardziej swiadczy o tym ze to tekst

Reply:
done